### PR TITLE
Use proper printf functions & refactor code

### PR DESCRIPTION
**Pull request by vishalkakadiya**
Ref: https://github.com/rtCamp/rtMedia/issues/1859#issue-1170810447

### CODE REVIEW

File: app/admin/RTMediaAdmin.php [hunk 1]
```diff
@@ -245,7 +245,12 @@ public function modify_medialibrary_permalink( $action, $post ) {
 				$title = _draft_or_post_title( $post->post_parent );
 
 				// translators: 1. Title.
-				$action['view'] = '<a href="' . esc_url( $link ) . '" title="' . esc_attr( sprintf( 'View "%s"', $title ) ) . '" rel="permalink">' . esc_html__( 'View', 'buddypress-media' ) . '</a>';
+				$action['view'] = sprintf(
+					'<a href="%1$s" title="%2$s" rel="permalink">%3$s</a>',
+					esc_url( $link ),
+					sprintf( 'View "%s"', $title ),
```
Comment (chandrapatel) [inline app/admin/RTMediaAdmin.php:251]: Escape it.

File: app/admin/RTMediaAdmin.php [hunk 2]
```diff
@@ -1637,11 +1674,16 @@ public function rtm_addon_license_notice() {
...
 
 				// translators: 1. Account page and link.
-				$message = sprintf( __( 'Your license keys can be found on <a href="%1$s">my-account</a> page. For more details, please refer to <a href="%2$s">License documentation</a> page.', 'buddypress-media' ), $my_account, $license_doc );
+				$message = sprintf(
+					__( 'Your license keys can be found on <a href="%1$s">my-account</a> page. For more details, please refer to <a href="%2$s">License documentation</a> page.', 'buddypress-media' ),
+					$my_account,
+					$license_doc
+				);
+
 				echo '<div class="notice"><p>' . wp_kses( $message, $args ) . '</p></div>';
```
Comment (chandrapatel) [inline app/admin/RTMediaAdmin.php:1687]: Use `printf()` here as well.

**Review by chandrapatel — APPROVED** [review]
Approved with minor feedback.

File: app/admin/RTMediaAdmin.php (hunk 1 above)
Comment (vishalkakadiya) [reply app/admin/RTMediaAdmin.php]: Done

File: app/admin/RTMediaAdmin.php (hunk 2 above)
Comment (vishalkakadiya) [reply app/admin/RTMediaAdmin.php]: Done

**Pull request merged**